src/Uno.UI.Composition/Generated/3.0.0.0/Windows.UI.Composition.Interactions/VisualInteractionSource.cs

```diff
@@ -232,13 +232,7 @@ public  float ScaleVelocity
 		// Forced skipping of method Windows.UI.Composition.Interactions.VisualInteractionSource.ScaleSourceMode.get
 		// Forced skipping of method Windows.UI.Composition.Interactions.VisualInteractionSource.ScaleSourceMode.set
 		// Forced skipping of method Windows.UI.Composition.Interactions.VisualInteractionSource.Source.get
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
```

Comment:
This is weird 🤔🤔🤔

Reply:
Fixed

Reply:
It's skipped in 

https://github.com/unoplatform/uno/blob/87084f8d6a5c258fd6f95ea5139449dbf36e0d10/src/Uno.UWPSyncGenerator/Generator.cs#L1175-L1183